src/static/css/cards.css

```diff
@@ -0,0 +1,37 @@
+.card-image{
+  display:block;
+  max-width:35% !important;
```

Comment:
I have no idea how to this without !important. :( I often struggle to overwrite the default CSS I have because of using materialize. But I think this styling is just temporary , I only added CSS to identify the problems we have to tackle when we make final design desisions. 

What's your advice ? How can I avoid using !important?